### PR TITLE
Add test file for DataFormats/L1ScoutingRawData 

### PR DESCRIPTION
**Pull request by Mmiglio**
Add initial data files that will be used in a unit test of the L1 Scouting raw data formats. These will be read by future releases to verify backward compatibility.

This is related to https://github.com/cms-sw/cmssw/pull/43467. I will open a new PR with improved unit tests for data format.

### CODE REVIEW

**Comment by smorovic** [comment]
please test

**Comment by smorovic** [comment]
+daq
(tests still running, but I guess new root file shouldn't in fact have effect)

**Comment by smuzaffar** [comment]
+externals

> These will be read by future releases to verify backward compatibility.

By the way, why not add the cmssw test now?

**Comment by Mmiglio** [comment]
Hi @cms-sw/orp-l2, is there anything missing for this PR to be merged?
I assume I need to wait for this PR to be merged before adding the test to CMSSW.
The same question applies to https://github.com/cms-data/DataFormats-L1Scouting/pull/1.

**Comment by smuzaffar** [comment]
This just contain a new data file, so I am merging it.

**Comment by smuzaffar** [comment]
@Mmiglio , note that cms-sw/opr-l2 is not a valid team for this organization

File: testSDSRawDataCollection_CMSSW_13_3_0_pre5.root
Comment (makortel) [inline testSDSRawDataCollection_CMSSW_13_3_0_pre5.root:1]: Also here, please include the class version in the file name, i.e. rename to `testSDSRawDataCollection_v3_CMSSW_13_3_0_pre5.root`.

**Pull request merged**